sass/index.scss

```diff
@@ -1,166 +1,632 @@
+
+        &.active {
+            font-weight: bold;
+            color: $black;
+            border-color: $black;
+
+            &:hover {
+                border-color: $black;
+            }
+        }
+    }
+
+    #logo {
+        width: 80px;
+        border: none;
+
+        img {
+            width: 100%;
+            height: 100%;
+        }
+    }
+}
+
+.mobile-grayout {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
     left: 0;
-    margin: 0 auto;
-    text-align: center;
+    background-color: black;
+    opacity: 25%;
+    z-index: 3;
+    cursor: pointer;
+    display: none;
+}
+
+// MLH Banner
+#mlh-trust-badge {
+    display: block;
+    position: fixed;
+    top: 0;
+    right: 5px;
+    width: 75px;
+    height: 0;
+    z-index: 2;
 }
 
-// Section styles
 
-#main {
+// Section padding
+section {
+    padding: 120px 5rem 0px; // Set up to avoid nav overlap when linked
+    min-height: 100vh;
     display: flex;
-    flex-direction: column;
+    flex-flow: column nowrap;
     justify-content: center;
-    align-items: center;
-    min-height: 100%;
-    // calc() seems to require var+const in that order
-    margin-top: calc((var(--logo-height) * -1) + var(--main-margin));
 }
 
-#logo {
-    width: var(--logo-height);
-    height: var(--logo-height);
-}
+// Hero
+#hero {
 
-#secret-message {
-    position: absolute;
-    margin-top: 50px;
-    z-index: 1;
 }
 
-.window {
-    max-width: 300px;
-    margin: var(--main-margin) auto;
-    position: relative;
-    z-index: 5;
+// About
+#about {
+    #about-content {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
 
-    .title-bar:hover {
-        cursor: grab;
+        #about-text {
+            width: 60%;
+        }
     }
-}
 
-.field-row-stacked {
-    input {
-        width: 150px;
-    }
+    #about-buttons {
+        display: flex;
+        flex-direction: column;
+        margin: 0px auto;
+        align-items: flex-start;
+
+        a {
+            @include button-primary;
+            display: flex;
+            align-items: center;
+            margin: 20px 0px;
+            width: 100%;
+        }
 
-    button {
-        margin: 0;
-        margin-left: 10px;
-        height: 21px; // Match the text box, hardcoded in the library :(
-        min-height: 0px;
+        i.fa-arrow-right, i.fa-images {
+            font-size: 1.6em;
+            color: $orange;
+            padding-right: 10px;
+        }
     }
 }
 
-#badges {
-    display: flex;
-    justify-content: center;
-    flex-wrap: wrap;
 
-    img {
-        margin: 0 calc(var(--main-margin) / 2);
-        background-color: #333;
-        height: 31px;
-        color: #333;
-        width: 88px;
+// FAQ
+#faq {
+    #faq-cards {
+        margin-top: 40px;
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-between;
+
+        // TODO: This can be cleaner.
+        $card-m: 20px;
+
+        &-left {
```

Comment:
This is cool! Haven't used SASS like this
Edit: after looking at the `git blame`, this FAQ code has been here since start of FAQ... im getting old